kahuna/public/js/services/image-accessor.js

```diff
@@ -0,0 +1,50 @@
+import angular from 'angular';
+
+const imageAccessor = angular.module('kahuna.services.image-accessor', []);
+
+/**
+ * Accessor helpers to read information out of image resources.
+ */
+imageAccessor.factory('imageAccessor', function() {
+
+    /* == Extractors ==  (return embedded Resources) */
+
+    function extractUserMetadata(image) {
+        return image.data.userMetadata;
+    }
+
+
+    /* == Readers ==  (return data) */
```

Comment:
That's how we named then in Composer, possibly to make it clearer it's a function and not a method on an object. I don't have a strong preference so happy to rename if others prefer getters?